ui/app/components/app/transaction-list-item/index.scss

```
@@ -31,4 +31,26 @@
       padding-top: 0;
     }
   }
+
+  .list-item__subheading > h3 {
+    overflow: visible;
+    display: flex;
+    white-space: nowrap;
+    text-overflow: initial;
+  }
+
+  .transaction-status:after {
+    content: "·";
+    margin: 0 4px;
+  }
+
+  &__origin, &__address {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+
+  &__origin {
+    direction: rtl;
```

```suggestion
    /*rtl:ignore*/
    direction: rtl;
```
We want this to be right-to-left even on right-to-left locales, but the build script to generate the rtl CSS file will flip this unless we tell it not to. This directive will ensure it stays as RTL for all locales